ftl/core/adding.ftl

```diff
@@ -1,5 +1,10 @@
 adding-add-shortcut-ctrlandenter = Add (shortcut: ctrl+enter)
 adding-added = Added
+adding-added-cards =
```

Comment:
Minor note: importing-cards-added can be used here instead of adding a new message. It's currently used for image occlusion on mobile.